src/Concerns/WithCustomQuerySize.php

```diff
@@ -0,0 +1,11 @@
+<?php
+
+namespace Maatwebsite\Excel\Concerns;
+
+interface WithCustomQuerySize
+{
+    /**
+     * @return int
+     */
+    public function count(): int;
```

Comment:
Can you name the method `querySize()`

Reply:
Perhaps also good to add some description to the docblock what it's meant for and what's expected to be returned

Reply:
Will do.

Reply:
Done!